apps/web/src/common/export.ts

```diff
@@ -33,7 +33,25 @@ export async function exportToPDF(
 
   return new Promise<boolean>((resolve) => {
     const iframe = document.createElement("iframe");
-    iframe.srcdoc = content.replaceAll(/<p(.+?)><\/p>/gm, "<p$1><br/></p>");
+
+    const replaceableAttributes = {
+      'data-align="right"':
+        'style="margin-left:auto;margin-right:0;display: block;"',
+      'data-align="center"':
+        'style="margin-left:auto;margin-right:auto;display: block;"',
+      'data-float="true" data-align="right"': 'align="right"',
+      'data-float="true" data-align="left"': 'align="left"'
+    };
+    const re = new RegExp(Object.keys(replaceableAttributes).join("|"), "gi");
+
+    let html = content.replaceAll(/<p(.+?)><\/p>/gm, "<p$1><br/></p>");
+    html = content.replaceAll(re, function (matched) {
```

Comment:
There's no need to use `RegExp` for this. Just loop over `replaceableAttributes` in a `for..in` loop and call `content.replaceAll`.